gradyrobbins/fourthStar
# Echo

# Description
User now able to see their own list of drills on index page upon login, rather than the entire list of drills.
User can add a drill and choose a dropdown list of categories for what type of drill it is.  This data is captured and INSERTed into DB.   



## Type of change

Please delete options that are not relevant.



# Testing Instructions for Change Made

Login under 2 separate accounts (admin + other acct) and create drills;  verify that each user is presented with their own list of drills created by themselves and no others.  Also can verify via SSMS SELECT * from DBO.Drills >> verify userId and categoryId columns have valid data

# Checklist:
